packages/terra-hookshot/src/Hookshot.jsx

```diff
@@ -0,0 +1,336 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Portal from 'react-portal';
+import HookshotContent from './_HookshotContent';
+import HookshotUtils from './_HookshotUtils';
+
+const ATTACHMENT_POSITIONS = [
+  'top left',
+  'top center',
+  'top right',
+  'middle left',
+  'middle center',
+  'middle right',
+  'bottom left',
+  'bottom center',
+  'bottom right',
+];
+
+const ATTACHMENT_BEHAVIORS = [
+  'auto',
+  'flip',
+  'none',
+];
+
+const propTypes = {
+  /**
+   * If the primary attachment in not available, how should the content be positioned.
```

Comment:
I think this would be worded better if we have the if statement at the end.

    How the content should be positioned when the primary attachment is not available.

Reply:
We need to list out the combinations